content/enterprise_features/sakuli-dashboard-configuration.md

```diff
@@ -148,7 +150,10 @@ The `CLUSTER_CONFIG` is required to connect to an existing cluster where you wan
 4. User to log onto cluster
 5. Namespace to execute actions in.
 
-### CRONJOB_CONFIG {#cronjob_config}
+**Important information**: A valid [CLUSTER_CONFIG](#cluster_config) is required, as soon as you want to apply actions using 
+[ACTION_CONFIG](#action_config), [DASHBOARD_CONFIG](#dashboard_config) and/or [CRONJOB_CONFIG](#cronjob_config) to a cluster.
```

Comment:
```suggestion
[ACTION_CONFIG](#action_config) and/or [CRONJOB_CONFIG](#cronjob_config) to a cluster.
```
I think we can omit `DASHBOARD_CONFIG` as it is not tied to applying actions to a cluster.